modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -1093,12 +1103,15 @@
 			</argument>
 			<argument index="1" name="to" type="float">
 			</argument>
-			<argument index="2" name="weight" type="float">
+			<argument index="2" name="x" type="float">
```

Comment:
This still says "x", did you forget to run doctool?

P.S. We should add CI checks for this eventually, I'll see if I can do it today.

Reply:
Right, sorry.
I recall now that internally after clamping `p_weight` between the edges, the original code called the clamped variable `x`. 
These commits changed that internal variable to `s` (and the parameter to `p_s`).